src/main/java/org/broadinstitute/hellbender/tools/picard/sam/GatherBamFiles.java

```diff
@@ -68,16 +68,14 @@ private boolean determineBlockCopyingStatus(final List<File> inputs) {
      * Simple implementation of a gather operations that uses SAMFileReaders and Writers in order to concatenate
      * multiple BAM files.
      */
-    private static void gatherNormally(final List<File> inputs, final File output, final boolean createIndex, final boolean createMd5,
+    private void gatherNormally(final List<File> inputs, final File output, final boolean createIndex, final boolean createMd5,
                                        final File referenceFasta) {
         final SAMFileHeader header;
         {
             header = SamReaderFactory.makeDefault().referenceSequence(referenceFasta).getFileHeader(inputs.get(0));
         }
 
-        try (final SAMFileWriter out =
-                     new SAMFileWriterFactory().setCreateIndex(createIndex).setCreateMd5File(createMd5).makeWriter(header, false, output, referenceFasta))
-        {
+        try (final SAMFileWriter out = createCommonSAMWriter(OUTPUT, REFERENCE_SEQUENCE, header, true)) {
```

Comment:
It does (via the common code), but the arg is no longer  needed in this method so I'll just remove it.